networking/eth2/src/testFixtures/java/tech/pegasys/teku/networking/eth2/Eth2P2PNetworkFactory.java

```diff
@@ -265,9 +261,13 @@ protected Eth2P2PNetwork buildNetwork(final P2PConfig config) {
                 spec,
                 NoOpKZG.INSTANCE,
                 __ -> Optional.of(discoveryNodeId),
-                dasTotalCustodySubnetCount,
                 DasReqRespLogger.NOOP);
 
+        if (spec.isMilestoneSupported(FULU)) {
+          metadataMessagesFactory.onCustodyGroupCountSynced(
+              SpecConfigFulu.required(spec.forMilestone(FULU).getConfig()).getCustodyRequirement());
```

Comment:
CGC count should probably come from manager now that it's persisted?

Reply:
it's test fixture